lightningd/peer_control.c

```diff
@@ -1499,6 +1501,53 @@ static const struct json_command listpeers_command = {
 /* Comment added to satisfice AUTODATA */
 AUTODATA(json_command, &listpeers_command);
 
+static struct command_result *json_listpeerchannels(struct command *cmd,
+						    const char *buffer,
+						    const jsmntok_t *obj UNNEEDED,
+						    const jsmntok_t *params)
+{
+	enum log_level *ll;
+	struct node_id *peer_id;
+	struct peer *peer;
+	struct channel *channel;
+	struct json_stream *response;
+
+	if (!param(cmd, buffer, params,
+		   p_req("id", param_node_id, &peer_id),
+		   p_opt("level", param_loglevel, &ll),
```

Comment:
id should be opt: default is all peers.  And level is not useful here...

Reply:
Ops! I grab the wrong history range of commits from the https://github.com/ElementsProject/lightning/pull/4821

This should be over with the latest changes.